libdokan/rekey_file.go

```diff
@@ -24,7 +24,9 @@ func (f *RekeyFile) WriteFile(ctx context.Context, fi *dokan.FileInfo, bs []byte
 	if len(bs) == 0 {
 		return 0, nil
 	}
-	err = f.folder.fs.config.KBFSOps().Rekey(ctx, f.folder.getFolderBranch().Tlf)
+	// TODO: get rid of this and use RequestRekey instead
```

Comment:
I think this needs to be synchronous, and it should return after the first rekey attempt either succeeds or fails.